src/lang/english.txt

```diff
@@ -3503,6 +3517,9 @@ STR_GROUP_PROFIT_THIS_YEAR                                      :Profit this yea
 STR_GROUP_PROFIT_LAST_YEAR                                      :Profit last year:
 STR_GROUP_OCCUPANCY                                             :Current usage:
 STR_GROUP_OCCUPANCY_VALUE                                       :{NUM}%
+STR_GROUP_EFFICIENCY_THIS_YEAR                                  :Efficiency this year:
+STR_GROUP_EFFICIENCY_LAST_YEAR                                  :Efficiency last year:
```

Comment:
We should do better, in-game, explaining what this number actually means. Via a tooltip for example. Efficiency can be read in so many ways.